src/app/login/login.component.html

```diff
@@ -0,0 +1,14 @@
+<form [formGroup]="loginForm" (ngSubmit)="onSubmit()">
+    
+    <label>
```

Comment:
make sure to also utilize the angular material components.

see: https://material.angular.io/components/form-field/overview